tests/test_web/test_api_v1.py

```diff
@@ -461,18 +452,30 @@ def test_add_bundle_from_index_and_add_arches_missing(mock_smfsc, db, auth_env,
     mock_smfsc.assert_not_called()
 
 
-@pytest.mark.parametrize('overwrite_from_index', (False, True))
+@pytest.mark.parametrize(
```

Comment:
Would it make sense to do this instead?
```python
@pytest.mark.parametrize('overwrite_from_index', (False, True))
@pytest.mark.parametrize('bundles', (['some:thing'], []))
@pytest.mark.parametrize('from_index', ('some:thing', None))
```

Then let pytest create the permutations.

Reply:
no because that can generate invalid permutations right?

Reply:
That's true. Ignore this.